docs/source/notes/ddp.rst

```diff
@@ -221,8 +221,8 @@ to fire in-between sections of backwards, and schedule communications to overlap
 
 See `this blog post <https://dev-discuss.pytorch.org/t/torchdynamo-update-9-making-ddp-work-with-torchdynamo/860/1>`_ for
 a more in-depth explanation and experimental results, or read the docs and code at
-`torch/_dynamo/optimizations/distributed.py <https://github.com/pytorch/pytorch/blob/4908a12542798a3e8641faae6b74f068fdfc6778/torch/_dynamo/optimizations/distributed.py#L56>`_
+`torch/_dynamo/optimizations/distributed.py <https://github.com/pytorch/pytorch/blob/bbc39b7bb48d28d67e3253a89cc82df3687ddd1b/torch/_dynamo/backends/distributed.py#L124>`_
 
-To Debug DDPOptimizer, set `torch._dynamo.config.log_level` to DEBUG (for full graph dumps) or INFO
+To Debug DDPOptimizer, set `TORCH_LOGS='+dynamo'` (for full graph dumps) or `TORCH_LOGS='dynamo'`
```

Comment:
actually there is a `ddp_graphs` log field which is probably the one you want to talk about here.  I'm not sure if adding `dynamo` to TORCH_LOGS does anything to the DDPOptimizer?

Reply:
Thanks! You're right.

The `ddp_graphs` seems to (only) print the graph, the bucket size is printed via `dynamo`:
https://github.com/pytorch/pytorch/blob/d5d62e85615fdf345e0556a9d8edbee2d3c64ae2/torch/_dynamo/backends/distributed.py#L13